static/app/views/explore/utils.tsx

```diff
@@ -69,6 +71,50 @@ export function getExploreUrl({
   );
 }
 
+export function getExploreMultiQueryUrl({
+  organization,
+  selection,
+  interval,
+  queries,
+  title,
+  id,
+}: {
+  interval: string;
+  mode: Mode;
+  organization: Organization;
+  queries: ReadableExploreQueryParts[];
+  selection: PageFilters;
+  id?: number;
+  title?: string;
+}) {
+  const {start, end, period: statsPeriod, utc} = selection.datetime;
+  const {environments, projects} = selection;
+  const queryParams = {
+    dataset: DiscoverDatasets.SPANS_EAP_RPC,
+    project: projects,
+    environment: environments,
+    statsPeriod,
+    start,
+    end,
+    interval,
+    queries: queries.map(({chartType, fields, groupBys, query, sortBys, yAxes}) =>
+      JSON.stringify({
+        chartType,
+        fields,
+        groupBys,
```

Comment:
This could be handled, but since I ran into it earlier with the Open in Explore flows from dashboards, if `groupBys` was an empty list, it wouldn't get pushed to the URL and then Explore was assuming it was supposed to get a default. I had to pin the groupBys to at least have an empty string. Just flagging in case it affects this or single query mode

Reply:
good call, I had to update this for single saved queries. Luckily the multi query view doesnt make any assumptions when groupby is empty